duckduckgo/macos-browser
# Bump privacy dashboard to latest version

Task/Issue URL: 
* https://app.asana.com/0/0/1202410371228723/f
* https://app.asana.com/0/1144193759412024/1202425934337331/f

Tech Design URL:
CC:

**Description**:

Bumps Privacy Dashboard to latest version -- this will fix the "Add tracker icon for Microsoft" issue within the privacy dashboard (as well as for a couple of other companies)




**Steps to test this PR**:
1. Build this branch
1. Go to https://www.workplace.com/ and open the Privacy Dashboard
1. Confirm the Microsoft tracker shows the company icon and not a generic "M" fallback icon


![](https://user-images.githubusercontent.com/635903/173037835-50779fdd-4986-4848-9b88-0dd7b54f357e.png)

**Testing checklist**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
